SavedInstances/Core/Core.lua

```diff
@@ -2514,6 +2514,43 @@ hoverTooltip.ShowDragonflightRenownTooltip = function (cell, arg, ...)
   finishIndicator()
 end
 
+hoverTooltip.ShowGrandHuntTooltip = function (cell, arg, ...)
+  -- Should be in Module Progress
+  local toon, index = unpack(arg)
+  local t = SI.db.Toons[toon]
+  if not t or not t.Progress or not t.Progress[index] then return end
+  if not t or not t.Quests then return end
+  openIndicator(2, "LEFT", "RIGHT")
+
+  local P = SI:GetModule("Progress")
+  local totalDone = 0
+  for _, questID in ipairs(P.TrackedQuest[index].relatedQuest) do
+    if t.Progress[index][questID] then
+      totalDone = totalDone + 1
+    end
+  end
+
+  local toonstr = (db.Tooltip.ShowServer and toon) or strsplit(' ', toon)
+
+  indicatortip:AddHeader(ClassColorise(t.Class, toonstr), string.format("%d/3", totalDone))
+
+  local naming = {
+    "|cffa335ee" .. "Epic" .. FONT_COLOR_CODE_CLOSE,
+    "|cff42a4f5" .. "Rare" .. FONT_COLOR_CODE_CLOSE,
+    "|cff1eff00" .. "Uncommon" .. FONT_COLOR_CODE_CLOSE,
```

Comment:
Use global string `ITEM_QUALITY4_DESC`, `ITEM_QUALITY3_DESC`, `ITEM_QUALITY2_DESC` for Epic, Rare and Uncommon. Also you can use `MAW_BUFF_QUALITY_STRING_EPIC`, `MAW_BUFF_QUALITY_STRING_RARE`, `MAW_BUFF_QUALITY_STRING_UNCOMMON` with color codes.

Reply:
I ended used MAW_BUFF_QUALITY_STRING but can't seem to pass the static check 

```
    SavedInstances/Core/Core.lua:14:38: accessing undefined variable 'MAW_BUFF_QUALITY_STRING_EPIC'
    SavedInstances/Core/Core.lua:15:38: accessing undefined variable 'MAW_BUFF_QUALITY_STRING_RARE'
    SavedInstances/Core/Core.lua:16:42: accessing undefined variable 'MAW_BUFF_QUALITY_STRING_UNCOMMON'
```

I tried to add on top of the file 
`local MAW_BUFF_QUALITY_STRING_EPIC = MAW_BUFF_QUALITY_STRING_EPIC` etc...
But I still get the error, any idea ?

Reply:
Thank you, everything should be fixed now